### PR TITLE
docs: Compile library CSS with docs

### PR DESCRIPTION
**Pull request by eboyer**
We have a couple broken references to CSS files. This should remove one unused CSS file as well as adding the gulp task `docs-css` to the `docs:compile` step.

### CODE REVIEW

**Comment by joseegm** [comment]
Test Version for this PR was deployed

Built with commit 8cec341b9dd9e3fc4e716936541c1fa26fea3e1c

https://deploy-preview-1378--fundamental.netlify.com

File: package.json [hunk 1]
```diff
@@ -7,7 +7,7 @@
     "build": "npx gulp build:dist",
     "build:prod": "npm run build -- --production",
     "deploy": "gh-pages -d docs/_site",
-    "docs:compile": "npm run docs:config && npm run build && npx gulp docs-site",
+    "docs:compile": "npm run docs:config && npm run build && npx gulp docs-build",
```
Comment (jbadan) [inline package.json]: we should be able to delete `npm run build` from here since `docs-build` calls `build:dist` same as in `npm run build`

**Comment by jbadan** [comment]
Do we need to add `docs/css/fundamental-ui.css` to the gitignore?

**Comment by eboyer** [comment]
@jbadan ~ I don't actually see that file being generated anywhere. I think it was just an artifact left around from awhile ago?

**Comment by jbadan** [comment]
It's being watched in `dev-watch` , maybe that should be watching `docs/css/fiori-fundamentals.css`?

**Comment by eboyer** [comment]
@jbadan and I have discussed addressing the `dev-watch` issues in a separate PR.

**Pull request merged**